Client/Application/AppDelegate.swift

```diff
@@ -244,7 +244,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(application: UIApplication, shouldAllowExtensionPointIdentifier extensionPointIdentifier: String) -> Bool {
-        return extensionPointIdentifier != UIApplicationKeyboardExtensionPointIdentifier
+        if let thirdPartyKeyboardSettingBool = getProfile(application).prefs.boolForKey("settings.allowThirdPartyKeyboards") where extensionPointIdentifier == UIApplicationKeyboardExtensionPointIdentifier {
```

Comment:
Nit: Since we use the string "settings.allowThirdPartyKeyboards" here and in the `AppSettingsTableViewController`, we can store it in a constant and refer to it? This way it will avoid any potential typo errors in the future if we have to type it out somewhere else.